staff/victor-abadia/cv/index.html

```diff
@@ -5,17 +5,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Nomen Nescio - Curriculum Vitae</title>
+    <link rel="stylesheet" type="text/css" href="style.css" media="all" />
 </head>
 <body>
-        <nav id="header">
+        <section id="header">
```

Comment:
AVOID using ids for identifying blocks / elements, do not use them if it's not strictly required by any other well-explained reason